### PR TITLE
added esConsumes to modules in EventFilter/DTRawToDigi

### PR DESCRIPTION
**Pull request by JamminJones**
#### PR description:

added esConsumes to 4 modules

#### PR validation:

the code compiles

### CODE REVIEW

**Comment by Dr15Jones** [comment]
This change was done under my supervision.

**Comment by Dr15Jones** [comment]
please test

**Review by slava77 — COMMENTED** [review]
this PR apparently includes #30162
please either close that one or update this one to decouple.
Thank you.

**Comment by Dr15Jones** [comment]
@slava77 thanks for pointing out the extra commit.

**Comment by Dr15Jones** [comment]
please test

File: EventFilter/DTRawToDigi/plugins/DTuROSRawToDigi.cc [hunk 1]
```diff
@@ -65,7 +64,7 @@ bool DTuROSRawToDigi::fillRawData(edm::Event& e,
   edm::Handle<FEDRawDataCollection> data;
   e.getByToken(Raw_token, data);
 
-  edm::ESHandle<DTReadOutMapping> mapping;
+  edm::ESHandle<DTReadOutMapping> mapping = c.getHandle(mapping_token_);
   c.get<DTReadOutMappingRcd>().get(mapping);
```
Comment (slava77) [inline EventFilter/DTRawToDigi/plugins/DTuROSRawToDigi.cc:68]:
```suggestion
  edm::ESHandle<DTReadOutMapping> mapping = c.getHandle(mapping_token_);
```

Comment (Dr15Jones) [reply EventFilter/DTRawToDigi/plugins/DTuROSRawToDigi.cc]: Thanks for the catch. We wanted to accept your change but the github interface we saw had the accept button but it was disabled. Hovering over the button said that your suggestion had no 'code change'. Guess we found a github bug.

**Comment by Dr15Jones** [comment]
please test

**Comment by slava77** [comment]
+1

for https://github.com/cms-sw/cmssw/pull/30166	08e5baf 
- technical code changes are in line with the PR description and the follow up review
- jenkins tests pass and comparisons with the baseline show no differences

**Comment by silviodonato** [comment]
+1

**Pull request merged**